### PR TITLE
podmanv2 pod inspect

### PR DESCRIPTION
**Pull request by sujil02**
Add the ability to inspect pod in podmanv2

Signed-off-by: Sujil02 <sushah@redhat.com>

### CODE REVIEW

**Comment by jwhonce** [comment]
@baude Should `podman inspect` be enhanced to call pod inspect now?

**Comment by mheon** [comment]
IMO, no - `docker inspect` doesn't do volumes, for example, last I checked.

**Comment by TomSweeneyRedHat** [comment]
LGTM assuming happy tests

File: cmd/podmanV2/pods/inspect.go [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+}
+
+func inspect(cmd *cobra.Command, args []string) error {
+
+	if len(args) < 1 && !inspectOptions.Latest {
+		return errors.Errorf("you must provide the name or id of a running pod")
+	}
+
+	if !inspectOptions.Latest {
+		topOptions.NameOrID = args[0]
```
Comment (vrothberg) [inline cmd/podmanV2/pods/inspect.go:52]:
```suggestion
		inspectOptions.NameOrID = args[0]
```

Comment (sujil02) [reply cmd/podmanV2/pods/inspect.go]: Fixed. Wasn't aware CI isn't in place. retested now works fine.

**Comment by vrothberg** [comment]
@rhatdan @baude @mheon PTAL

File: cmd/podmanV2/pods/inspect.go [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+}
+
+func inspect(cmd *cobra.Command, args []string) error {
+
+	if len(args) < 1 && !inspectOptions.Latest {
+		return errors.Errorf("you must provide the name or id of a running pod")
+	}
+
+	if !inspectOptions.Latest {
+		inspectOptions.NameOrID = args[0]
```
Comment (mheon) [inline cmd/podmanV2/pods/inspect.go:52]: If we're just discarding other arguments, should we ensure that there is at most 1 argument passed?

File: pkg/domain/entities/pods.go [hunk 3]
```diff
@@ -3,6 +3,7 @@ package entities
 import (
 	"time"
 
+	"github.com/containers/libpod/libpod"
```
Comment (mheon) [inline pkg/domain/entities/pods.go:6]: @baude We OK with this?

Comment (baude) [reply pkg/domain/entities/pods.go:6]: no, i will follow up with you and sujil

**Pull request merged**